tower_modules/tower_inventory_source/tasks/main.yml

```diff
@@ -14,21 +13,36 @@
 - name: Add a Tower inventory
   tower_inventory:
     description: Test inventory
-    name: openstack-test-inventory
     organization: Default
+    name: openstack-test-inventory
 
 - name: Create a source inventory
   tower_inventory_source:
-    name: "source-test-inventory {{ lookup('randstr') }}"
+    name: "source-test-inventory"
```

Comment:
This is probably better to leave as a randomly generated name, if you want to delete it later you can store it as a variable before this task or use the result variable to get the name

Reply:
Ok, I'll revert this change... I couldn't get `randstr` to work locally because it's a `test-playbook` specific thing and I forgot to change it back.